tests/publiccloud/run_ltp.pm

```diff
@@ -48,11 +48,11 @@ sub instance_log_args
 sub upload_ltp_logs
 {
     my ($self) = @_;
-    my $log_file = Mojo::File::path('ulogs/ltp_log.json');
+    my $log_file = Mojo::File::path('ulogs/result.json');
     my $ltp_testsuite = get_required_var('LTP_COMMAND_FILE');
 
-    upload_logs("$root_dir/ltp_log.raw", log_name => 'ltp_log.raw', failok => 1);
-    upload_logs("$root_dir/ltp_log.json", log_name => $log_file->basename, failok => 1);
+#    upload_logs("$root_dir/ltp_log.raw", log_name => 'ltp_log.raw', failok => 1);
```

Comment:
what about raw result ? they don't have it anymore ?

Reply:
raw results were really only a stdout redirection. You can use tee command and bring stdout if you need it

Reply:
@m-dati please remove this line